tendermint/src/lite/verifier.rs

```diff
@@ -1,24 +1,34 @@
-#[allow(clippy::all)]
-use crate::lite::{Commit, Error, Header, Validator, ValidatorSet, ValidatorSetLookup, Vote};
-use crate::Time;
-use std::time::Duration;
+//! Main verification functions that can be used to implement a light client.
+//!
+//!
+//! # Examples
+//!
+//! ```
+//! // TODO: add a proper example maybe showing how a `can_trust_bisection`
+//! // looks using the types and methods in this crate/module.
+//!```
+
+use crate::lite::{Commit, Error, Header, SignedHeader, TrustThreshold, ValidatorSet};
+use std::time::{Duration, SystemTime};
 
 /// Returns an error if the header has expired according to the given
 /// trusting_period and current time. If so, the verifier must be reset subjectively.
 /// NOTE: this doesn't belong here. It should be called by something that handles whether to trust
```

Comment:
Think this comment might be stale?

Reply:
removed in https://github.com/interchainio/tendermint-rs/pull/84/commits/a855701a649dcc097b7e1c66cf52bbe01374978e